packages/db/src/lib/granule.ts

```diff
@@ -354,3 +354,58 @@ export const getGranulesByGranuleId = async (
     .where({ granule_id: granuleId });
   return records;
 };
+
+/**
+ * Change a granules' PG record and its files' PG record based on collection move
+ * 
+ * @param {Knex | Knex.Transaction} knexOrTransaction - DB client or transaction
+ * @param {Object} [collectionPgModel] - Collection PG model class instance
+ * @param {string[]} granuleIds - list of granules by granuleIds to change
+ * @param {string} collectionId - collection ID
+ * @returns {Promise<void>}
+ */
+ export const updateGranuleAndFiles = async (
+  knexOrTransaction: Knex | Knex.Transaction,
+  collectionPgModel = new CollectionPgModel(),
+  granuleIds: string[],
```

Comment:
I think I'd like to be giving you full granules that you can get files and collection_id from
I also have and can give you source_granules and target_granules so you have a before and after state